tests/golos.publication_tests.cpp

```diff
@@ -155,6 +155,10 @@ class golos_publication_tester : public golos_tester {
         const string no_social_acc         = amsg("Social account doesn't exist.");
         const string no_referral_acc       = amsg("Referral account doesn't exist.");
         const string not_valid_ref_block_num = amsg("ref_block_num mismatch");
+        const string wrong_prmlnk_length   = amsg("Permlink length doesn't validated.");
+        const string wrong_prmlnk          = amsg("Permlink doesn't match with naming convension.");
+        const string wrong_title_length    = amsg("Title length doesn't validated.");
+        const string wrong_body_length     = amsg("Body length doesn't validated.");
```

Comment:
Error messages should help the user understand what the problem is. This text is too general. It is better to describe the error in more detail. For example: for body field it will be better: `Body is empty`